playbooks/roles/matrix/tasks/k8s.yaml

```diff
@@ -0,0 +1,177 @@
+---
+- name: Create matrix Namespace
+  community.kubernetes.k8s:
+    context: "{{ context }}"
+    state: "present"
+    name: "{{ namespace }}"
+    api_version: "v1"
+    kind: "Namespace"
+
+- name: Create matrix ConfigMap
+  community.kubernetes.k8s:
+    context: "{{ context }}"
+    state: "{{ state }}"
+    namespace: "{{ namespace }}"
+    name: "{{ matrix_cm_name }}"
+    kind: "ConfigMap"
+    api_version: "v1"
+    definition:
+      metadata:
+        labels:
+          app.kubernetes.io/name: "matrix"
+          app.kubernetes.io/instance: "{{ instance }}"
+          app.kubernetes.io/managed-by: "system-config"
+      data:
+        homeserver.yaml: "{{ lookup('template', 'homeserver.yaml.j2') }}"
+        signing.key: "{{ lookup('template', 'signing.key.j2') }}"
+        log.config: "{{ lookup('template', 'log.config.j2') }}"
+
+- name: Create matrix TLS Certs Secret
```

Comment:
dont we need apply:yes here too ? In case the cert is renewed, the secret needs to be updated too

Reply:
somehow this moment was never properly tested (what happens with ingress when secret is updated). Anyway added